1.9/usage/pods/pods-cli.md

```diff
@@ -0,0 +1,49 @@
+---
+post_title: Pods CLI
+menu_order: 20
+---
+
+You can create and manage your pods via the DC/OS CLI. The following commands are available:
+
+* dcos marathon pod add [<pod-resource>]
+* dcos marathon pod list [--json]
+* dcos marathon pod remove [--force] <pod-id>
+* dcos marathon pod show <pod-id>
+* dcos marathon pod update [--force] <pod-id> [<properties>...]
+
+# Add a Pod
+
+To add a pod, first create a JSON pod definition. Then, run the following command:
+```
+dcos marathon pod add <mypod.json>
```

Comment:
Unless `<mypod.json>` is supposed to represent _any_ file argument? If that's the case then `<pod-resource>`, `<pod-json-file>`, or something similar might be better? Having the `.json` suffix made me think this was referring to an actual file name.